pkg/tsdb/sqleng/resample/resample.go

```diff
@@ -90,10 +91,7 @@ func resample(f *data.Frame, qm dataQueryModel) (*data.Frame, error) {
 	lastSeenRowIdx := -1
 	timeField := f.Fields[tsSchema.TimeIndex]
 
-	startUnixTime := qm.TimeRange.From.Unix() / int64(qm.Interval.Seconds()) * int64(qm.Interval.Seconds())
-	startTime := time.Unix(startUnixTime, 0)
```

Comment:
here we remove the align-time-range-start code (we add it back in `sql_engine.go`)